packs/mssql/actions/execute.query.yaml

```diff
@@ -0,0 +1,33 @@
+---
+pack: mssql
+name: execute.query
+description: Queries MS SQL Server and writes the results to CSV. Returns list of paths to CSV files.
+enabled: true
+
+runner_type: python-script
+entry_point: mssql_runner.py
+
+parameters:
+  action:
+    default: execute_query
+    immutable: true
+    type: string
+  query_string:
+    description: Query to send to MS SQL Server.
+    required: true
+    type: string
+  params:
+    description: Parameters for interpolation in the query_string.
+    type: string
```

Comment:
I think that I actually started with an object. We lose some flexibility by using an object though, unless I just preprocess the object back into a format like this.

The README might make this make more sense.

```python
    query_string = "SELECT * FROM empl WHERE id=%d"
    params = 13
    
    query_string = "SELECT * FROM empl WHERE name=%s"
    params = "'John Doe'"
    
    query_string = "SELECT * FROM empl WHERE name LIKE %s"
    params = "'J%'"
    
    query_string = "SELECT * FROM empl WHERE name=%(name)s AND city=%(city)s"
    params = "{ 'name': 'John Doe', 'city': 'Nowhere' }"
    
    query_string = "SELECT * FROM cust WHERE salesrep=%s AND id IN (%s)"
    params = "('John Doe', (1, 2, 3)))"
```

For what its worth, `ast.literal_eval` [is always safe](https://docs.python.org/2/library/ast.html#ast.literal_eval), unlike `eval`.